tests/test_intopydict.rs

```diff
@@ -0,0 +1,32 @@
+use pyo3::{prelude::IntoPyDict, types::PyDict, Python};
```

Comment:
I think this will fix the CI failure.

```suggestion
#![cfg(feature = "macros")]
use pyo3::{prelude::IntoPyDict, types::PyDict, Python};
```

Reply:
Thank you so much for your reply! The reason I made this contribution is because I had to manually implement IntoPyDict on some of my rust structs even though the implementation seemed relatively trivial (ie only primitive fields). I figured there should be a derive macro for the trivial case. I am passing the output into an tf model, and I find it is easier to parse the elements from a dictionary to a tensor on the python side.

Reply:
Also I'm having a little bit of trouble with the quote! macro. Could you help me out with it?

Reply:
I can't say I am an expert on it. I'd recommend reading `quote`'s documentation, online guides and our existing macro code to see what it does.

Reply:
Do you agree with the use case? Also why did the CI stop running checks. This is my first time committing to an open source repo like this.

Reply:
> Do you agree with the use case?

The other maintainers would have to weigh in as well, but I think I can see where you are coming from. An example would help.

You can just run the tests locally, these should catch most issues.

Reply:
In the local tests it keeps on failing on the test 'test_compile_errors'